### PR TITLE
Add project arg to ert vis launcher

### PR DESCRIPTION
**Pull request by ManInFez**
Launch ert storager server if no server is found

### CODE REVIEW

File: ert_shared/plugins/__init__.py [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
 )
+from ert_shared.storage.connection import get_info, get_project_id
+from ert_shared.storage.server_monitor import ServerMonitor
 
 
 def launch_visualization_plugin(args):
+
+    # Try to use project specified in args (Defaults to cwd)
+    try:
+        get_info(args.project)
```
Comment (xjules) [inline ert_shared/plugins/__init__.py:15]: Can this be handled in a more pythonic way?
```python
if bool(args.project):
 ...
else:
  ...
```

File: ert_shared/storage/connection.py [hunk 2]
```diff
@@ -58,3 +58,16 @@ def set_global_info(project_path: Union[str, Path]):
...
+
+
+def get_project_id() -> Path:
+    """Return a path to `storage_server.json`."""
+
+    if "XDG_RUNTIME_DIR" in os.environ:
+        project_path = Path(os.environ["XDG_RUNTIME_DIR"]) / "ert"
+    else:
+        project_path = Path(f"/tmp/ert-{getpass.getuser()}")
+    json_path = project_path / "storage_server.json"
```
Comment (xjules) [inline ert_shared/storage/connection.py:70]: ```os.path.join(...)``` ?

File: ert_shared/plugins/__init__.py (hunk 1 above)
Comment (oysteoh) [reply ert_shared/plugins/__init__.py:15]: the test is not about args.project beeing `None` i guess - get_info is raising Exception if no connection

**Comment by oysteoh** [comment]
Everything seems to work as expected but running with -p / --project. It starts without any issues - but are not showing data. Are you picking up the arg and using this correctly? I cannot figure out how it is picked up and used at least 😬

File: ert_shared/storage/connection.py (hunk 2 above)
Comment (jondequinor) [reply ert_shared/storage/connection.py:70]: we should use `pathlib`, see e.g. https://treyhunner.com/2018/12/why-you-should-be-using-pathlib/

**Review by oysteoh — APPROVED** [review]
🚀

**Comment by oysteoh** [comment]
remember to squash the two commits 👍

**Pull request merged**